src/main/webapp/admin_dashboard.html

```diff
@@ -1,127 +1,93 @@
-									<div class="admin-task-infos">
-										<p class="date"><i class="far fa-calendar"></i> 06/30/2020</p>
-										<p class="time"><i class="far fa-clock"></i> 4:00pm</p>
-									</div>
-								</li>
-								<li class="admin-task">
-									<h3>Upload Tasks</h3>
-									<h4>Leonard Zhang</h4>
-									<div class="admin-task-infos">
-										<p class="date"><i class="far fa-calendar"></i> 06/30/2020</p>
-										<p class="time"><i class="far fa-clock"></i> 4:00pm</p>
-									</div>
-								</li>
-							</ul>
-						</div>
-					</div>
-					<div id="chart-container">
-						<h2>Usage Statistics</h2>
-						<div id="chart-div"></div>
-					</div>
-				</div>
-				<div class="flex-grid">
-					<div id="map-container">
-						<h2>Map</h2>
-						<div id="map-div"></div>
-					</div>
-					<div class="card-container">
-						<h2> <span>Last 10 User Tasks</span></h2>
-						<div id="card-div">
-							<ul id="user-tasks">
-							</ul>
-						</div>
-					</div>
-				</div>
-			</main>
-		</div>
-		<div id="popup-overlay">
-			<div id="popup">
-				<h2>Edit Task</h2>
-				<form action="/admin-user-tasks" method="POST">
-					Task Details: <input type="text" id="task-detail-input" name="task-detail-input">
-					Reward: <input type="text" id="reward-input" name="reward-input" min="0" max="200">
-					<select name="edit-category-input" id="edit-category-input">
-						<option value="garden">Garden</option>
-						<option value="shopping">Shopping</option>
-						<option value="pets">Pets</option>
-						<option value="misc">Misc</option>
-					</select>
-                    <input type="submit" value="Submit">
-					<input type="hidden" name="task-id" id="task-id-input">
-				</form>
-				<a href="#" id="popup-close">&times;</a>
-                
-			</div>
-		</div>
-	</body>
-</html>
+<!DOCTYPE html>
+<!--
+	Copyright 2020 Google LLC
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+-->
+
+<html>
+	<head>
+		<title>Admin Dashboard</title>
+		<link rel="stylesheet" href="admin_dashboard_style.css">
+		<script src='https://kit.fontawesome.com/a076d05399.js'></script>
+		<script src="https://www.gstatic.com/charts/loader.js"></script>
+		<script type="text/javascript" src="config.js"></script>
+		<script src="admin_dashboard_script.js"></script>
+	</head>
+	<body>
+		<div class="wrapper">
+			<main>
+				<h1><a href="/"><i class="fas fa-user-circle fa-2x" title="Go to User Page" aria-hidden="true"></i></a> Admin Dashboard</h1>

```

Comment:
Currently the user button will take the users back to the main page instead of user profile page, which is a little bit frustrating. Although I think it is a good idea to take the users back to the main page, we should have a different icon and add some padding to the top. Currently the icon is kinda too close to the top.